handlers/mapping.go

```diff
@@ -0,0 +1,54 @@
+package handlers
+
+import dpHTTP "github.com/ONSdigital/dp-net/http"
```

Comment:
Not sure what's right, but the casing of this alias doesn't look right. I'd have gone with `dphttp` but I'm not sure if that is right either.